### PR TITLE
doc: Update LICENSE and Readme file for project [skip ci]

### LINKED ISSUE #138: Change LICENSE file of the root folder
Should explain that there are Apache and CVAL licensed components

### PR DESCRIPTION
**Pull request by ZheSun88**
Fix #138

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ This is a maven multi-module project including all vaadin flow components.
 
 `master` branch is the latest version of all the components that will be released in the [Vaadin platform](https://github.com/vaadin/platform).
 
+## LICENSE
+SPDX-License-Identifier: Apache-2.0 OR Commercial Vaadin Add-on License version 3
+
+This project is dual-licensed under [Apache 2.0](http://www.apache.org/licenses/LICENSE-2.0) and [Commercial Vaadin Add-on License version 3](https://vaadin.com/license/cval-3.0) (CVALv3).
```
Comment (manolo) [inline README.md:10]: I would list the components under each license here somethning like ç

This project is dual-licensed, thus the license that applies depends on the component:
- [Apache 2.0](http://www.apache.org/licenses/LICENSE-2.0) for core components (vaadin-grid, vaadin ....)
- [Commercial Vaadin Add-on License version 3](https://vaadin.com/license/cval-3.0)  for pro components (vaadin-crud ... )

File: LICENSE [hunk 2]
```diff
@@ -1,8 +1,6 @@
-All parts, except the contents of the documentation module, are licenced
-under Apache License v2.0. See the license text below.
-
-The documentation is licensed under Creative Commons CC-BY-ND 2.0
-(http://creativecommons.org/licenses/by-nd/2.0/legalcode).
+This project dual-licensed under Apache 2.0, for free components, and
```
Comment (manolo) [inline LICENSE:1]: Having 2 licenses in the file is confusing, I would just write a sentence with : please visit the license file in each module, or the same text than in the readme file

**Review by manolo — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ZheSun88) [reply README.md:10]: the reason why i don't want to list all the components there is that I don't want to maintain the README file in the future, when we have modified one new/old component, also considering different branches have different component set. 

i think ``

File: LICENSE (hunk 2 above)
Comment (ZheSun88) [reply LICENSE]: done

**Review by manolo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (manolo) [reply README.md:10]: Done

**Pull request merged**